code/game/machinery/computer/manager_camera.dm

```diff
@@ -305,7 +346,21 @@ GLOBAL_VAR_INIT(execution_enabled, FALSE)
 
 	if(istype(clicked_atom, /mob/living/simple_animal))
 		var/mob/living/simple_animal/monster = clicked_atom
+		if (user.mind.assigned_role == "Manager" && GetFacilityUpgradeValue(UPGRADE_ARCHITECT_2))
+			var/message = "<span class='notice'>[clicked_atom]'s current health is : [monster.health] \n[clicked_atom]'s resistances are :"
```

Comment:
This is entirelly on whoever did the code below, but the span could be applied smarter
Instead of opening it and then closing it at the end, try making this just a string and adding `span_notice(message)` to your `to_chat` at the end